app/dockerdwrapper.c

```diff
@@ -622,17 +621,17 @@ static bool start_dockerd(const struct settings* settings, struct app_state* app
                            G_SPAWN_DO_NOT_REAP_CHILD | G_SPAWN_SEARCH_PATH,
                            NULL,
                            NULL,
-                           &dockerd_process_pid,
+                           &rootlesskit_pid,
                            &error);
     if (!result) {
         log_error("Starting dockerd failed: execv returned: %d, error: %s", result, error->message);
         set_status_parameter(param_handle, STATUS_NOT_STARTED);
         goto end;
     }
-    log_debug("Child process dockerd (%d) was started.", dockerd_process_pid);
+    log_debug("Child process rootlesskit (%d) was started.", rootlesskit_pid);
 
     // Watch the child process.
-    g_child_watch_add(dockerd_process_pid, dockerd_process_exited_callback, app_state);
+    g_child_watch_add(rootlesskit_pid, dockerd_process_exited_callback, app_state);
```

Comment:
is it worth renaming `dockerd_process_exited_callback` as well?

Reply:
I would like to rename it and also move it to a place where no forward declaration is necessary, but that will be a separate PR.